content/sensu-core/1.4/reference/events.md

```diff
@@ -122,6 +122,12 @@ possible values | `create`, `resolve`, `flapping`
 default         | `create`
 example         | {{< highlight shell >}}"action": "create"{{< /highlight >}}
 
+last_ok       | 
+--------------|-----
+description   | The time of the most recent check execution indicating 'OK' status for this client/check pair (generated via Ruby `Time.now.to_i`).
```

Comment:
Is there a simpler way to say `check execution`? Would `check result` still be accurate?

To break up the noun cluster, you could try something like: The most recent time a check result indicated an ‘OK’ status for this client/check pair.